db/migrate/20190711085633_drop_pgde_course.rb

```diff
@@ -0,0 +1,9 @@
+class DropPGDECourse < ActiveRecord::Migration[5.2]
+  def up
+    drop_table :pgde_course
+  end
+
+  def down
```

Comment:
Could make it a `change` rather than `up/down`

Reply:
```
drop_table(without block) is not reversible.
```